src/restore.c

```diff
@@ -789,18 +847,24 @@ restore_chain(pgBackup *dest_backup, parray *parent_chain,
 	}
 
 	/* Get list of files in destination directory and remove redundant files */
-	if (params->incremental_mode != INCR_NONE)
+	if (params->incremental_mode != INCR_NONE || cleanup_pgdata)
```

Comment:
Add a comment about the 'cleanup_pgdata' parameter.

Reply:
Sure